gui/themes/common/highres_layout.stx

```diff
@@ -2314,6 +2314,15 @@
 					width = '480'
 					height = '250'
 			/>
+			<layout type = 'horizontal' padding = '-80, 8, 8, 8'>
```

Comment:
That's unusually high negative padding. How does it look like?

Reply:
![scummvm-football2002-00007](https://user-images.githubusercontent.com/5963757/230338981-6d58f90c-d015-45ca-80d5-19da49f7cf7a.png)

This was done so that the label and edit boxes would properly appear to the left of the dialog instead of the center.  And for comparision, here's with the default 8 padding:

![scummvm-football2002-00008](https://user-images.githubusercontent.com/5963757/230340716-9ff051ab-bda9-4669-97d7-8a6de4c67130.png)

Actually, now that I think about it, it doesn't make too much of a difference, I'll revert the padding back to 8.

Reply:
Edited and force pushed the themes commit with padding now set to 8.